sdv/multi_table/base.py

```diff
@@ -336,8 +358,8 @@ def fit_processed_data(self, processed_data):
             processed_data (dict):
                 Dictionary mapping each table name to a preprocessed ``pandas.DataFrame``.
         """
-        augmented_data = self._augment_tables(processed_data)
-        self._model_tables(augmented_data)
+        augmented_data = self._augment_tables(processed_data.copy())
+        self._model_tables(augmented_data, prefix='\n')
```

Comment:
the base `_model_tables` doesn't have the new prefix arg, so this would crash on HSA right?

Reply:
Nice catch, let me add it :shipit: